packages/rainbowkit/src/wallets/walletConnectors/imToken/imToken.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { WalletConnectConnector } from 'wagmi/connectors/walletConnect';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { rpcUrlsForChains } from '../../../utils/rpcUrlsForChains';
+import { Wallet } from '../../Wallet';
+
+export interface ImTokenOptions {
+  chains: Chain[];
+}
+
+export const imToken = ({ chains }: ImTokenOptions): Wallet => ({
+  id: 'imToken',
+  name: 'imToken',
+  iconUrl: async () => (await import('./imToken.svg')).default,
+  iconBackground: 'transparent',
+  downloadUrls: {
+    android: 'https://play.google.com/store/apps/details?id=im.token.app',
+    ios: 'https://itunes.apple.com/us/app/imtoken2/id1384798940',
+    qrCode: 'https://token.im/',
```

Comment:
```suggestion
    qrCode: 'https://token.im/download',
```

@izayl other apps provide a link that will automatically pick the appropriate app store for the platform (i.e., Rainbow uses `https://rainbow.download` which takes you to the App Store on an iOS device). Any idea if imToken has one of these? In it's absence, the link I've provided might be better still.